openedx/core/djangoapps/notifications/base_notification.py

```diff
@@ -230,6 +230,22 @@
         'email_template': '',
         'filters': [FILTER_AUDIT_EXPIRED_USERS_WITH_NO_ROLE],
     },
+    'audit_access_expiring_soon': {
+        'notification_app': 'enrollments',
+        'name': 'audit_access_expiring_soon',
+        'is_core': True,
```

Comment:
We are in the process of deprecating core notification feature because that was created for an edge case that was abandoned. Please update this to False and add default values in notification type instead of app level config.

Reply:
ok